src/js/components/DateInput/DateInput.js

```diff
@@ -140,7 +159,7 @@ const DateInput = forwardRef(
                 }
               }
         }
-        {...calendarProps}
+        {...{ ...calendarProps, timestamp }}
```

Comment:
```suggestion
        { ...calendarProps, timestamp }
```

Reply:
I was getting an error when trying to spread like this